lib/cc/engine/csslint.rb

```diff
@@ -52,6 +27,35 @@ def run
 
       private
 
+      def create_issue(node, path)
+        begin
+          check_name = node.attributes.fetch("identifier").value
+          check_details = CheckDetails.fetch(check_name)
+          return {
```

Comment:
Some style nits: We don't add explicit `return`s as they're not needed in Ruby. Please put a line of whitespace before the (implicit) return value so set it apart from the body of the method.